desktop/test/lib/setup.coffee

```diff
@@ -0,0 +1,137 @@
+sinon = require 'sinon'
+rewire = require 'rewire'
+express = require 'express'
+path = require 'path'
+
+describe 'setup', ->
+
+  before ->
+    @setup = rewire '../../lib/setup'
+
+  describe 'setup production environment', ->
+
+    before ->
+      @setup.__set__
+        NODE_ENV: 'production'
+        MAX_SOCKETS: 10
+        ipfilter: @ipfilter = sinon.stub().returns((req, res, next) -> next())
+        artsyPassport: @artsyPassport = sinon.stub().returns((req, res, next) -> next())
+        session: @session = sinon.stub().returns((req, res, next) -> next())
+        bodyParser:
+          json: @json = sinon.stub().returns((req, res, next) -> next())
+          urlencoded: @urlencoded = sinon.stub().returns((req, res, next) -> next())
+        bucketAssets: @bucketAssets = sinon.stub().returns((req, res, next) -> next())
+        cors: @cors = sinon.stub().returns((req, res, next) -> next())
+        artsyPassport: @artsyPassport = sinon.stub().returns((req, res, next) -> next())
```

Comment:
Not sure which editor you use, but you can install CoffeeLint directly into your editor (independent of a project-specific lintfile) to catch things like this -- thats how this jumped out :)